contrib/opencensus-ext-azure/opencensus/ext/azure/common/__init__.py

```diff
@@ -17,14 +17,73 @@
 
 from opencensus.ext.azure.common.protocol import BaseObject
 
+AUTHORIZATION = 'Authorization'
+DEFAULT_BREEZE_ENDPOINT = 'https://dc.services.visualstudio.com'
```

Comment:
We should avoid codename here.

Reply:
To clarify, is this because `DEFAULT_BREEZE_ENDPOINT` is a module level variable?